pkgs/applications/misc/bambu-studio/default.nix

```diff
@@ -28,6 +28,7 @@
   mesa,
   mpfr,
   nlopt,
+  slic3r,
```

Comment:
I think believe this is doing much, since it's never used anywhere. I also think bambu studio ships their own version, so it might be error prone to override

Reply:
Understood. Will remove it